mikeio/spatial/grid_geometry.py

```diff
@@ -635,8 +644,12 @@ def _xy_to_index(self, xy):
         y = xy[:, 1]
         x = xy[:, 0]
 
-        ii = (-1) * np.ones_like(x, dtype=int)
-        jj = (-1) * np.ones_like(y, dtype=int)
+        ii = (-1) * np.ones_like(
+            x, dtype=int
+        )  # TODO -1 is not a good choice, since it is a valid index
+        jj = (-1) * np.ones_like(
+            y, dtype=int
+        )  # TODO -1 is not a good choice, since it is a valid index
```

Comment:
Should we replace all the -1 with -999999999 for now? It's certainly not optimal - but it will not be a valid index...